import-export-cli/integration/testutils/environment_testUtils.go

```diff
@@ -411,3 +411,234 @@ func validateMutualSSLCerts(t *testing.T, apiParams *Params, path string) {
+	importedAPI.SetProductionURL(same)
+	importedAPI.SetSandboxURL(same)
+
+	ValidateAPIsEqual(t, api, importedAPI)
+}
+
+func ValidateHttpEndpointWithLoadBalancing(t *testing.T, api *apim.API, apiParams *Params, importedAPI *apim.API) {
+	t.Helper()
+	//Validate EndPoint Type
+	assert.Equal(t, "load_balance", importedAPI.GetEndpointType())
+
+	endPointConfigInApi := importedAPI.EndpointConfig
+
+	// Validate Algorithm class
+	algoClassNameInParams := apiParams.Environments[0].Configs.LoadBalanceEndpoints.AlgorithmClassName
+	algoClassNameInApi := endPointConfigInApi.(map[string]interface{})["algoClassName"].(string)
+	assert.Equal(t, algoClassNameInParams, algoClassNameInApi)
+
+	// Validate Session TimeOut
+	sessionTimeOutInParams := apiParams.Environments[0].Configs.LoadBalanceEndpoints.SessionTimeout
+	sessionTimeOutInApi := endPointConfigInApi.(map[string]interface{})["sessionTimeOut"].(string)
+	assert.Equal(t, strconv.Itoa(sessionTimeOutInParams), sessionTimeOutInApi)
+
+	// Validate Production endpoints
+	productionEndpointsInParams := apiParams.Environments[0].Configs.LoadBalanceEndpoints.Production
+	productionEndpointsInApi := endPointConfigInApi.(map[string]interface{})["production_endpoints"].([]interface{})
+	if strings.EqualFold(apiParams.Environments[0].Configs.LoadBalanceEndpoints.SessionManagement, "transport") {
+		for i := 0; i < len(productionEndpointsInParams); i++ {
+			singleProductionEndpointInApi := productionEndpointsInApi[i].(map[string]interface{})
+			assert.Equal(t, productionEndpointsInParams[i]["url"], singleProductionEndpointInApi["url"])
+		}
+	}
+	if strings.EqualFold(apiParams.Environments[0].Configs.LoadBalanceEndpoints.SessionManagement, "soap") {
+		for i := 0; i < len(productionEndpointsInParams); i++ {
+			singleProductionEndpointInApi := productionEndpointsInApi[i].(map[string]interface{})
+			assert.Equal(t, productionEndpointsInParams[i]["url"], singleProductionEndpointInApi["url"])
+			assert.Equal(t, "address", singleProductionEndpointInApi["endpoint_type"])
+		}
+	}
+
+	// Validate Sandbox endpoints
+	sandboxEndpointsInParams := apiParams.Environments[0].Configs.LoadBalanceEndpoints.Sandbox
+	sandboxEndpointsInApi := endPointConfigInApi.(map[string]interface{})["sandbox_endpoints"].([]interface{})
+	if strings.EqualFold(apiParams.Environments[0].Configs.LoadBalanceEndpoints.SessionManagement, "transport") {
+		for i := 0; i < len(sandboxEndpointsInParams); i++ {
+			singleSandboxEndpointInApi := sandboxEndpointsInApi[i].(map[string]interface{})
+			assert.Equal(t, sandboxEndpointsInParams[i]["url"], singleSandboxEndpointInApi["url"])
+		}
+	}
+	if strings.EqualFold(apiParams.Environments[0].Configs.LoadBalanceEndpoints.SessionManagement, "soap") {
+		for i := 0; i < len(sandboxEndpointsInParams); i++ {
+			singleSandboxEndpointInApi := sandboxEndpointsInApi[i].(map[string]interface{})
+			assert.Equal(t, sandboxEndpointsInParams[i]["url"], singleSandboxEndpointInApi["url"])
```

Comment:
Any reason for not having,
assert.Equal(t, "address", singleSandboxEndpointInApi["endpoint_type"])
as you did for production endpoints?

If we add this code line this will be similar to 485 to 491. We can move this to a separate function.